modules/ROOT/pages/managing-instances/instance-details.adoc

```diff
@@ -54,6 +54,30 @@ A snapshot is a copy of the data in an instance at a specific point in time.
 Neo4j regularly takes snapshots of your instance, and you can request a snapshot to be taken on demand.
 These snapshots can be used to restore data to a different Neo4j instance.
 
+== Vector optimization
+
+label:AuraDB-Professional[]
+label:AuraDB-Business-Critical[]
+label:AuraDB-Virtual-Dedicated-Cloud[]
+
+Vector optimization reserves memory for vector indexes, enhancing performance for vector-based operations.
+It is available for AuraDB instances with more than 4GB of memory and across all supported cloud providers and regions.
+
+This configuration re-allocates memory from the graph database to the vector index. 
+If your application needs better vector and graph performance, you should resize to a larger Aura instance.
```

Comment:
```suggestion
This configuration re-allocates memory to the vector index at the expense of non-vector query performance. 
If this has a negative impact on your application, consider resizing your Aura instance.
```
I don't think it is correct to say that the vector index and the graph database are two separate things; the vector index, like any other index, is part of the graph database. Additionally, we need to avoid trying to upsell our users, at least in the docs.